documentation/docs/libraries/python/how_to/_09_get_block/_expected_output.md

```diff
@@ -0,0 +1,38 @@
+```json
```

Comment:
Out of curiosity, could you try `js` instead and see if it changes anything? The highlighting should stay the same but GitHub might stop paining everything red. If it works, we could use `js` for `json` snippets as a general rule to spare our eyes on the long run.

Reply:
We can, but we should do that in a different PR